tests/test_utils.py

```diff
@@ -122,6 +123,7 @@ def test_logging_redirect_chain(server, caplog):
     ]
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="os path sep escaped on windows")
```

Comment:
We don't need this `skipif` now, right?

Reply:
we still need this skipif because path seprator `/` is double escaped on this tests, don't know why....

Reply:
Can you remove the `skipif`, so we can see the exact error in CI?

Reply:
```text
E
E         At index 1 diff: ('httpx', 10, "load_verify_locations cafile='C:\\\\Users\\\\Trim21\\\\proj\\\\httpx\\\\.venv\\\\Lib\\\\site-packages\\\\certifi\\\\cacert.pem'") != ('httpx', 10, "load_verify_locations cafile='C:\\Users\\Trim21\\proj\\httpx\\.venv\\Lib\\site-packages\\certifi\\cacert.pem'")
E
E         Full diff:
E           [
E               (
E                   'httpx',
E                   10,
E                   'load_ssl_context verify=True cert=None trust_env=True http2=False',
E               ),
E               (
E                   'httpx',
E                   10,
E                   'load_verify_locations '
E         -         "cafile='C:\\Users\\Trim21\\proj\\httpx\\.venv\\Lib\\site-packages\\certifi\\cacert.pem'",
E         +         "cafile='C:\\\\Users\\\\Trim21\\\\proj\\\\httpx\\\\.venv\\\\Lib\\\\site-packages\\\\certifi\\\\cacert.pem'",
E         ?                      ++     ++          ++    ++         ++       ++   ++                 ++       ++
E               ),
E           ]

```